daq/faucet_states_collector.py

```diff
@@ -66,53 +80,82 @@ def get_switches(self):
             switch_data[switch_name] = self.get_switch(switch_name)
         return switch_data
 
+    def get_switch_map(self):
+        """returns switch map for topology overview"""
+        switch_map = {}
+        topo_obj = self.topo_state
+        with self.lock:
+            for switch in topo_obj.get(TOPOLOGY_GRAPH, {}).get("nodes", []):
+                _id = switch.get("id")
```

Comment:
_ unnecessary/redundant -- should just be id

Reply:
id is a reserved name in python. id causes lint to crib about it.